bin/docker-build.js

```diff
@@ -938,15 +1129,20 @@ function zfsSnapshotStream(opts, callback, onProcessError) {
     var stderr = '';
     var zoneUuid = opts.zoneUuid;
 
+    var snapshotFullname = util.format('zones/%s@%s', zoneUuid, snapshot);
     var zfsBase = sprintf('zones/%s', zoneUuid);
     var args = [
         '-r', 'root',                         // Include zone root directory
         '-x', 'native',                       // Exclude anything in /native/
         '-x', 'checkpoints',                  // Exclude snapshot /checkpoints/
         '-x', 'var/svc/provision_success',    // Exclude vm provisioning file
-        '-x', 'var/log/sdc-dockerinit.log',   // Exclude sdc docker log file
-        zfsBase, parent_snapshot, snapshot
+        '-x', 'var/log/sdc-dockerinit.log'    // Exclude sdc docker log file
     ];
+    if (opts.commandType === 'commit') {
```

Comment:
I'm ignorant of the reason why we differentiate these two cases, maybe a comment would help any other reader not familiar with this code base :)